src/store/createStore.ts

```diff
@@ -1,13 +1,21 @@
-import { Store } from 'redux';
 import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit';
+import API from '../api';
 import createRootReducer from './createRootReducer';
 import getEventingMiddleware from './eventing';
-import { ApplicationState } from './types';
+import { AppState, AppStore } from './types';
+
+export type Options = {
+    api?: API;
```

Comment:
Should this be required?

Reply:
Making it required will complicate unit tests quite a bit. It's always defined at runtime, so it seems mostly dogmatic to make it required here.